webservices/rest.py

```diff
@@ -732,7 +732,15 @@ class Candidate(object):
 
     def query_text(self, show_fields):
         if show_fields['cmte_fields'] != '':
-            com_query = "/dimlinkages?cmte_dsgn={%s}{*, /dimcmte{/dimcmteproperties{cmte_nm}}}  :as affiliated_committees," % (show_fields['cmte_fields'])
+            cmte = show_fields['cmte_fields']
+            if cmte == 'P':
+                cmte_des_qry = "?cmte_dsgn={'P'}"
```

Comment:
{'P"} - for future reference, the curly braces aren't necessary when you're comparing to a single value; their only purpose is to let you do an OR-type comparison to multiple values.  Still, I don't think they do any harm so there's no need to remove them.